src/Synth/OscilGen.cpp

```diff
@@ -16,8 +16,11 @@
 #include "../DSP/FFTwrapper.h"
 #include "../Synth/Resonance.h"
 #include "../Misc/WaveShapeSmps.h"
+#include "../Params/WaveTable.h"
+
 
 #include <cassert>
+#include <chrono>
```

Comment:
At least with a quick CTRL+f it looks like this and the timer_t class are currently unused.

Reply:
Correct. I used that in the past, but it is not required anymore. Will be removed in next commit.